.test-infra/jenkins/job_LoadTests_Combine_Java.groovy

```diff
@@ -0,0 +1,216 @@
+                    jobProperties: [
+                            project             : 'apache-beam-testing',
+                            appName             : "load_tests_Java_Dataflow_${jobName}_Combine_3",
+                            tempLocation        : 'gs://temp-storage-for-perf-tests/loadtests',
+                            publishToBigQuery   : true,
+                            bigQueryDataset     : 'load_test',
+                            bigQueryTable       : "java_dataflow_${jobName}_Combine_3",
+                            sourceOptions       : """
+                                                    {
+                                                      "numRecords": 2000,
+                                                      "keySizeBytes": 100000,
+                                                      "valueSizeBytes": 900000
+                                                    }
+                                               """.trim().replaceAll("\\s", ""),
+                            fanout              : 1,
+                            iterations          : 1,
+                            topCount            : 20,
+                            maxNumWorkers       : 5,
+                            numWorkers          : 5,
+                            autoscalingAlgorithm: "NONE",
+                            perKeyCombiner      : "TOP_LARGEST",
+                            streaming           : streaming
+                    ]
+
+            ],
+            [
+                    title        : 'Load test: fanout 4 times with 2GB 10-byte records total',
+                    itClass      : 'org.apache.beam.sdk.loadtests.CombineLoadTest',
+                    runner       : CommonTestProperties.Runner.DATAFLOW,
+                    jobProperties: [
+                            project             : 'apache-beam-testing',
+                            appName             : "load_tests_Java_Dataflow_${jobName}_Combine_4",
+                            tempLocation        : 'gs://temp-storage-for-perf-tests/loadtests',
+                            publishToBigQuery   : true,
+                            bigQueryDataset     : 'load_test',
+                            bigQueryTable       : "java_dataflow_${jobName}_Combine_4",
+                            sourceOptions       : """
+                                                    {
+                                                      "numRecords": 5000000,
+                                                      "keySizeBytes": 10,
+                                                      "valueSizeBytes": 90
+                                                    }
+                                               """.trim().replaceAll("\\s", ""),
+                            fanout              : 4,
+                            iterations          : 1,
+                            topCount            : 20,
+                            maxNumWorkers       : 5,
```

Comment:
According to proposal numer of runners changes to 16.

Reply:
Changed according to the proposal, thanks!